### PR TITLE
clarify global access token summary

### PR DESCRIPTION
**Pull request by adeola-ak**
## Test plan
Doc update

### CODE REVIEW

File: doc/batch_changes/how-tos/configuring_credentials.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Interacting with a code host (such as creating, updating, or syncing changesets)
 There are two types of access token that can be configured for use with Batch Changes:
 
 1. [**Personal access token**](#personal-access-tokens) - A token set by an individual Batch Changes user for their personal code host user account.
-1. [**Global service account token**](#global-service-account-tokens) (*Admins only*) - A token that can be used by any Batch Changes user who does not have a personal access token configured. These are also required for [importing changesets](./tracking_existing_changesets.md) and syncing changeset state from the code host when webhooks are not configured.
+1. [**Global service account token**](#global-service-account-tokens) (*Configurable by admins only*) - A token that can be used by any Batch Changes user who does not have a personal access token configured. These are also required for [importing changesets](./tracking_existing_changesets.md) and syncing changeset state from the code host when webhooks are not configured. If you’re concerned about individual user permissions, we would not advise using a global access token.
```
Comment (eseliger) [inline doc/batch_changes/how-tos/configuring_credentials.md:15]: The problem is that without it we don't support changeset syncing today. :( 

cc @malomarrec I think we should add this ASAP for strategic.

**Review by eseliger — APPROVED** [review]

File: doc/batch_changes/how-tos/configuring_credentials.md (hunk 1 above)
Comment (chrispine) [reply doc/batch_changes/how-tos/configuring_credentials.md:15]: Can you create a ticket for this, @eseliger (or move one into Triage if we have one already) and give a rough estimate?

Comment (malomarrec) [reply doc/batch_changes/how-tos/configuring_credentials.md:15]: I recommend adding this with priority strategic/medium (importing changesets is not used in practice).

**Review by malomarrec — APPROVED** [review]

File: doc/batch_changes/how-tos/configuring_credentials.md (hunk 1 above)
Comment (adeola-ak) [reply doc/batch_changes/how-tos/configuring_credentials.md:15]: should i add "Although currently importing changesets is not supported without a global access token, if you’re concerned about individual user permissions, we would not advise using a global access token." or leave it the way it is? @eseliger

Comment (eseliger) [reply doc/batch_changes/how-tos/configuring_credentials.md:15]: I can live with it either way, you decide :) 

Filed https://github.com/sourcegraph/sourcegraph/issues/42490

**Review by BolajiOlajide — APPROVED** [review]

**Pull request merged**